perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -84,6 +87,30 @@ public Member call(ApiCaller ac, Deserializer parms) throws PerunException {
 			}
 		}
 	},
+	
+	/**
+	 * Creates a new SPONSORED member from given parameters map.
+	 *
+	 * @param parameters Map<String,String> Map containing all parameters, must contain at least:
+	 *                   "sponsor" - uco of sponsor for new created account
+	 * @param namespace String namespace
+	 * @param extSource int extSource ID
+	 * @param extSourcePostfix String extSource login postfix, e.g. "@muni.cz"
+	 * @param vo int VO ID
+	 * @return Member newly created sponsored member
+	 */
+	createSponsoredAccount {
```

Comment:
I know returning ``Member`` is aligned with the rest of our API methods for member creation. But I believe INET expectation is, that they will receive generated UČO in a response (returning param map with necessary attributes would be better). I'm letting this to @licehammer to decide. INET will create sponsored members in batches and they probably want to keep it simple.